ClientInfoPlugin.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+namespace DNWS
+{
+  class ClientInfoPlugin : IPlugin
+  {
+    protected static Dictionary<String, int> statDictionary = null;
```

Comment:
Why we need statDictionary thingy inside ClientInfoPlugin?

Don't just make your code work, make it perfect!!!!